internal/command/httpserver.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/spf13/cobra"
 	"go.uber.org/zap"
 
-	"github.com/elastic/stream/pkg/httpserver"
+	"github.com/elastic/stream/internal/pkg/httpserver"
```

Comment:
```suggestion
	"github.com/elastic/stream/internal/httpserver"
```
?
(throughout)

Reply:
💯

I’m glad you caught that, because that is what I wanted. I’m not a good multi-tasker.